.github/workflows/code_review.yml

```diff
@@ -0,0 +1,22 @@
+name: AI Code Reviewer
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+permissions: write-all
+jobs:
+  review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
+      - name: AI Code Reviewer
+        uses: lfsevergnini/ai-codereviewer@chore/add-model-version-parameter
```

Comment:
The AI Code Reviewer action has been updated to use a different repository. Please make sure to test the changes thoroughly before merging.